conf/application.conf

```diff
@@ -37,6 +37,11 @@ microservice {
         host     = localhost
         port     = 8500
       }
+      digital-services-tax {
+        protocol = http
+        host     = localhost
+        port     = 8741
+      }
```

Comment:
do we need to add `digital-services-tax` to other Envs, such as `app-config-base`?

Reply:
I'll raise a PR for this, thanks!

Reply:
https://github.com/hmrc/app-config-base/pull/11290